src/time.rs

```diff
@@ -62,6 +67,13 @@ impl Instant {
     pub fn total_millis(&self) -> i64 {
         self.millis
     }
+
+    /// Return the [Duration] between two [Instant's].
+    ///
+    /// [Duration] struct.Duration.html
+    pub fn duration_since(&self, other: Instant) -> Duration {
```

Comment:
Shouldn't this be `impl Sub<Instant> for Instant`?

Reply:
👍